devpack/src/main/java/io/neow3j/devpack/contracts/PolicyContract.java

```diff
@@ -135,6 +137,50 @@ public PolicyContract() {
     @CallFlags(All)
     public native void setStoragePrice(int price);
 
+    /**
+     * @return the block generation time in milliseconds
+     */
+    @CallFlags(ReadOnly)
+    public native int getMillisecondsPerBlock();
+
+    /**
+     * Sets the block generation time in milliseconds.
+     *
+     * @param milliseconds The block generation time in milliseconds.
+     */
+    @CallFlags(States | AllowNotify)
+    public native void setMillisecondsPerBlock(int milliseconds);
+
+    /**
+     * @return the upper increment size of blockchain height (in blocks)
+     * exceeding that a transaction should fail validation.
```

Comment:
nitpick: please use the default line lenght as in the rest of the file (same for the javadoc of `setMaxValidUntilBlockIncrement`).